x-pack/plugin/core/src/main/java/org/elasticsearch/license/XPackLicenseState.java

```diff
@@ -225,7 +225,10 @@ private static String[] enterpriseSearchAcknowledgementMessages(OperationMode cu
                     case TRIAL:
                     case PLATINUM:
                     case ENTERPRISE:
-                        return new String[] { "Search Applications and behavioral analytics will be disabled" };
+                        return new String[] {
+                            "Search Applications and behavioral analytics will be disabled.",
```

Comment:
How will these be displayed? Is it better to have one long message detailing what will be disabled, such as in `monitoringAcknowledgementMessages`?

Reply:
https://github.com/elastic/kibana/pull/179614 Here is the Kibana PR to give an example. Our use case is similar to `securityAcknowledgementMessages` rather